packages/mongo-nav/src/org-nav/OrgNav.spec.tsx

```diff
@@ -53,6 +53,7 @@ describe('packages/mongo-nav/src/org-nav', () => {
     expectedElements.billing = queryByTestId('org-billing');
     expectedElements.allClusters = queryByTestId('all-clusters-link');
     expectedElements.admin = queryByTestId('admin-link');
+    expectedElements.version = queryByTestId('onPrem-version');
```

Comment:
[nitpick] this should be snake-cased as `on-prem-version`

Reply:
Also having just written that last comment for the review, it might be nice to prefix the testids with the component name, so they're namespaced. I think @DesignerDave was trying to surreptitiously get me to do that in a previous PR.

Reply:
👍